oop/IM_Project/src/device.cpp

```diff
@@ -0,0 +1,35 @@
+#include "device.h"
+
+Device::Device(const Sensor& sensor, SensorType sensor_type, int number_binding, const Date& calibration_date)
+    : mSensor(sensor), mSensorType(sensor_type), mNumberBinding(number_binding), mCalibrationDate(calibration_date){} 
```

Comment:
add space between ) and {
```suggestion
    : mSensor(sensor), mSensorType(sensor_type), mNumberBinding(number_binding), mCalibrationDate(calibration_date) {} 
```